content/post/targeted-quantiles.md

```diff
@@ -0,0 +1,160 @@
++++
+title = "Targeted Quantiles in Prometheus"
+date = "2018-03-11T19:25:53Z"
+draft = false
++++
+
+_The algorithm behind Prometheus' Summaries._
+
+## Abstract
+
+Prometheus uses a technique called Targeted Quantiles to implement the Summary metric type which is used to monitor distributions, such as latencies.
+Here, we give an overview of how the algorithm works, we answer why it is so efficient and how to configure Summaries.
+
+## Introduction
+
+A very important way of understanding how your distributed system works in production is to monitor it.
+In recent years, Prometheus has emerged as the most popular monitoring system for applications deployed on Kubernetes.
+
+Apart from the more common counters and gauges, the Prometheus client exposes the [summary](https://godoc.org/github.com/prometheus/client_golang/prometheus#Summary) metric type.
+In a go project, it might look like this:
+
+```golang
+// This is the definition of a metric which tracks the latency of an RPC request.
```

Comment:
Should I know what RPC is?

Reply:
I think this is common knowledge in distributed systems theory.